### PR TITLE
Tidy up json tests

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

File: solution/Koan/Parser/Json/Test.hs [hunk 1]
```diff
@@ -20,6 +18,9 @@ import qualified Koan.Parser.Json     as K
 {-# ANN module ("HLint: ignore Redundant do"        :: String) #-}
 {-# ANN module ("HLint: ignore Reduce duplication"  :: String) #-}
 
+enrolled :: Bool
+enrolled = True
```
Comment (Kheldar) [inline solution/Koan/Parser/Json/Test.hs]: Probably not intentional

**Comment by Kheldar** [comment]
Did we want to save the solutions anywhere?

**Comment by AlexeyRaga** [comment]
👍

File: solution/Koan/Parser/Json/Test.hs (hunk 1 above)
Comment (newhoggy) [inline solution/Koan/Parser/Json/Test.hs]: This one is because it's in the solutions. 😁

**Pull request merged**